website/docs/docs/cloud/dbt-cloud-ide/lint-format.md

```diff
@@ -53,12 +53,13 @@ Linting doesn't support ephemeral models in dbt v1.5 and lower. Refer to the [FA
 
 ### Enable linting
 
-1. To enable linting, make sure you're on a development branch. Linting isn't available on main or read-only branches.
-2. Open a `.sql` file and click the **Code Quality** tab.
-3. Click on the **`</> Config`** button on the bottom right side of the [console section](/docs/cloud/dbt-cloud-ide/ide-user-interface#console-section), below the **File editor**. 
-4. In the code quality tool config pop-up, you have the option to select **sqlfluff** or **sqlfmt**. 
-5. To lint your code, select the **sqlfluff** radio button. (Use sqlfmt to [format](#format) your code)
-6. Once you've selected the **sqlfluff** radio button, go back to the console section (below the **File editor**) to select the **Lint** or **Fix** dropdown button:
+Linting is available on all branches, including your protected primary git branch. When ready, you can commit those changes to a new branch.
```

Comment:
```suggestion
Linting is available on all branches, including your protected primary git branch. Since the dbt Cloud IDE prevents commits to the protected branch, you can commit those changes to a new branch.
```